Makefile

```diff
@@ -255,12 +254,13 @@ legends: $(VENV) guard-BODID guard-WMSHOST
 
 
 .PHONY: serve
-serve: $(VENV) config-templates set-app_version
+serve: setup config-templates set-app_version
+	pipenv install -e .
```

Comment:
This should be part of the setup target as before, this will also solve the CI test issue.